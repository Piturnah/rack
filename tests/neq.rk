// Should get `0 1 1 0`
false false != print
false true != print
true false != print
true true != print

// Should get nothing
5 5 != if
  13 print
end

// Should get 100
5 4 != if
  100 print
end