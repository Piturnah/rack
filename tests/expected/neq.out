----STDOUT----
0
1
1
0
100

----STDERR----
